### PR TITLE
libass

### PR DESCRIPTION
**Pull request by scopatz**
hopefully this works

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@scopatz 
Should we merge it?

**Comment by scopatz** [comment]
CC @conda-forge/staged-recipes

File: recipes/libass/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - autoconf
+    - automake
+    - pkg-config
+    - libtool
+  host:
+    - nasm
```
Comment (isuruf) [inline recipes/libass/meta.yaml]: Should this be in `build`?

Comment (scopatz) [reply recipes/libass/meta.yaml]: I think the package might use nasm to generate assembly for each platform separately.

Comment (scopatz) [reply recipes/libass/meta.yaml]: ie it doesn't cross-compile

**Pull request merged**